.github/workflows/tag-release.yaml

```diff
@@ -0,0 +1,111 @@
+---
+name: Release on tag ⬆
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.REPO_GITHUB_TOKEN }}
+    permissions:
+      contents: write
+      packages: write
+      deployments: write
+    steps:
+
+    - name: Checkout code 🛎
+      uses: actions/checkout@v2      
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Get body from tag msg 🔍
+      run: |
+        # set TAG_MESSAGE
+        COMMIT_MESSAGE=$(git log --format=%B -n 1 ${{ github.ref }})
+        TAG_MESSAGE=$(git tag -l --format='%(contents:subject)' ${{ github.ref }})
+        echo "TAG_MESSAGE=${TAG_MESSAGE}" >> $GITHUB_ENV
+      shell: bash
+    
+    - name: Create Release ${{ github.ref }} 🚀
+      uses: actions/create-release@v1
```

Comment:
Use an alternative action. This one is no longer maintained: https://github.com/actions/create-release#github-action---releases-api